proto/api/v1/process_whitelist_service.proto

```diff
@@ -1,96 +0,0 @@
-syntax = "proto3";
```

Comment:
Was a changelog ever published for this removal?

Reply:
I'm not sure I understand. The file removed are for Process **Whitelist** service which has been deprecated since 55. Process **Baseline** are still around